### ISSUE TITLE
Not able to parse string[]

### ISSUE THREAD

**Issue by UmashankarSharma** [JamesRandall/FunctionMonkey#88]
Hi James,

I am giving something like below in my model class. but compiler shows the below issue.

    public class GetCaseEntitiesQuery : ICommand<ErrorResponse>
        {

        [JsonProperty("ids")]
        public string[] Ids { get; set; }

        }
    }

**Unhandled Exception: FunctionMonkey.ConfigurationException: CS0117:'string' does not contain a definition for 'TryParse' SourceFile**

**Comment by JamesRandall** [comment]
Could you try making that a List<string> please and let me know if that helps please?
The query parameters don't currently work with basic arrays. Its on my list :)

**Comment by UmashankarSharma** [comment]
Hi James,

You are quick :) . I wanted to do that but Unfortunately , I could not change the contract of API's :(.

Thanks a lot.

Uma

**Comment by JamesRandall** [comment]
You caught me at my email :)

Changing it to a list won't change the external contract (I assume that's what you mean) - and if you're passing the ids down to some internal service via a command handler you should be able to do a .ToArray on it.

**Comment by UmashankarSharma** [comment]
Thanks, 
James. Let me try it.

**Comment by UmashankarSharma** [comment]
The "**List<string> Ids { get; set; }**" did not work either... But I got another way of doing that.

**public dynamic Ids { get; set; }**

This works.

Thanks a lot, James.

**Comment by CodeRevver** [comment]
I'm getting this issue as well.  I'm on the latest beta

**Comment by JamesRandall** [comment]
Hi @CodeRevver - do you mean with an array?

I've just rerun the integration tests and the scenario I'm expecting to work is working. That is with a command like the following:

```csharp
public class HttpListQueryParamCommand : ICommand<int>
{
    public List<int> Value { get; set; }
}
```

If you have a different variant on this could you post it here for me please! See if I can get to the bottom of this as it sounds like there is something I might be missing.

**Comment by CodeRevver** [comment]
@JamesRandall   I've reproduced it here: 

- https://github.com/CodeRevver/function-monkey-string-list-bug

Thanks for getting back to us so quickly - it's appreciated.

**Comment by JamesRandall** [comment]
Thanks for that - I see what's wrong now. I'll get a fix out shortly.

Thanks both for reporting it and helping me get to the bottom of it!

**Comment by JamesRandall** [comment]
I've resolved this and pushed an update to Nuget v3.0.13-beta5.

Thanks again for your help.
